app/Exceptions/Handler.php

```diff
@@ -94,11 +95,6 @@ class Handler extends ExceptionHandler
      */
     public function report(Throwable $exception)
     {
-        if (! Schema::hasTable('accounts')) {
-            info('account table not found');
-            return;
-        }
-
         if (Ninja::isHosted()) {
```

Comment:
did you want to place this after the missingappkeyexception so that it also logs the output?

Reply:
I think it's better to remove it completely.  The problem is if the .env file is missing or the database credentials are incorrect then Schema::hasTable throws an error rather than return false. Since this code is inside the error handler it causes the execution to stop immediately making it harder to debug problems.

Reply:
There are edge cases which if this block is not included, then /setup will be inaccessible.

Happy to merge, but there is a reason the block is there.

Reply:
Can you provide steps to replicate? In my tests this code blocked me from initially accessing the /setup route. As explained above you'll see a 500 error in the app but no logs to troubleshoot the problem. 

I think in general relying on the database inside the global exception handler isn't a great idea.

Reply:
sure, copy the .env.example to .env and then attempt /setup without that block (the assumption here is that the database credentials are either incorrect, or the database does not exist).

Without a similar type of handler, the database exception will never allow the setup screen to be displayed. That is the reason why it is necessary to escape inside the handler.

Reply:
That's not correct, /setup loads fine for me.

It takes longer than I think it should take to load but I believe that's a separate issue which I plan to look into.

Reply:
if it works for you, great! i'll merge.

Reply:
Thanks, I think this is an important change. I'm trying to test my changes as much as I can to reduce any headaches for you.

Next PR incoming...